code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -163,10 +163,10 @@
 /obj/item/reagent_containers/blood/attackby(obj/item/attacking_item, mob/user)
 	..()
 	if (attacking_item.ispen())
-		if (REAGENT_VOLUME(reagents, /singleton/reagent/blood) && name != "empty blood pack") //Stops people mucking with bloodpacks that are filled
+		if (REAGENT_VOLUME(reagents, /singleton/reagent/blood) && name != "empty IV bag") //Stops people mucking with bloodpacks that are filled
 			to_chat(user, SPAN_NOTICE("You can't relabel [name] until it is empty!"))
 			return
-		var/blood_name = tgui_input_list(user, "What blood type would you like to label it as?", "Blood Types",  list("A+", "A-", "B+", "B-", "O+", "O-", "AB+", "AB-", "Saline Plus", "Clear", "Cancel"))
+		var/blood_name = tgui_input_list(user, "What would you like to label the IV bag?", "Label Selection",  list("A+ Blood", "A- Blood", "B+ Blood", "B- Blood", "O+ Blood", "O- Blood", "AB+ Blood", "AB- Blood", "Saline Plus", "Cryonics Mixture", "Other Mixture", "Clear", "Cancel"))
```

Comment:
The proper way to do this at this point is to use singletons to determine what types of blood bags there are, their icon state, and their "type" (so if it started as an empty bag, ...)

Reply:
im not sure where to start approaching this tbh or how it would look in contrast to what's already done - besides being a lot neater lol